argparse_test.go

```diff
@@ -441,7 +501,7 @@ func TestFailCaseSensitive(t *testing.T) {
 	testArgs := []string{"progname", "-F"}
 
 	p := NewParser("", "description")
-	_ = p.Flag("f", "", &Options{Required: true})
```

Comment:
Tries to test case sensitivity, but actually - the undefined argument check because of silent argument drop

Reply:
as far as I remember long name should be required one and short name is optional. that should be in godoc or in readme.

Reply:
Yeah. And here only a shortened argument was provided.